extensions/azure/storage/AzureDataLakeStorage.h

```diff
@@ -24,11 +24,12 @@
 #include <memory>
 #include <optional>
 #include <utility>
+#include <string_view>
 
 #include "core/logging/Logger.h"
 #include "core/logging/LoggerConfiguration.h"
 #include "DataLakeStorageClient.h"
-#include "azure/core/io/body_stream.hpp"
+#include "utils/ListingStateUtils.h"
```

Comment:
Updated in ef0172d119c472a1c91a10aecbe531a112ade2bf

I agree, usually I try to avoid names like *Manager, *Handler and so on, but I think in this case it is okay as the `ListingStateManager` is already a specific use case of the `CoreComponentStateManager` so at least there is a connection in the names.